src/controller/AutoCommissioner.cpp

```diff
@@ -247,7 +247,7 @@ CommissioningStage AutoCommissioner::GetNextCommissioningStageInternal(Commissio
         // TODO(cecille): device attestation casues operational cert provisioning to happen, This should be a separate stage.
         // For thread and wifi, this should go to network setup then enable. For on-network we can skip right to finding the
         // operational network because the provisioning of certificates will trigger the device to start operational advertising.
-        if (mNeedsNetworkSetup)
+        if (mParams.GetNeedsNetworkSetup().ValueOr(false))
```

Comment:
Isn't this changing behavior for existing non-chiptool consumers?  We can't really do that by default.

Reply:
Setting this value to true when the `TransportType` is `kBle`.